packages/members/theme-dropin/src/index.js

```diff
@@ -0,0 +1,51 @@
+const DomReady = require('domready');
+const GhostContentApi = require('@tryghost/content-api');
+const GhostMembersLayer2 = require('@tryghost/members-layer2');
+
+const fetchAndLoadContent = (hostUrl, postId, token) => {
+    const postContentDiv = document.querySelector('[data-members-drop-in-resource');
+    if (!postContentDiv) {
+        return;
+    }
+    const api = GhostContentApi.create({
+        host: hostUrl,
+        version: 'v2',
+        key: ''
+    });
+
+    api.posts.read({ id: postId }, {}, token).then((post) => {
```

Comment:
Should this be `api[postContent.getAttribute('data-members-drop-in-resource')].read` ?

Reply:
Yeah can be, didn't add as it felt like pre-optimizing since its all built around only post resource for now so left it for later when case comes, though did use this pattern in initialization so we have structure of how it works. Open go either ways here 😄 🤷‍♂️

Reply:
Yeah fair enough :smile: Let's remove it from the casper changes if it doesn't exist yet though i rekon?

Reply:
hmm `data-members` seems more appropriate. but lets just get this done and merged, can clean up after

Reply:
^ 💯 agreed!